webapp/views/App/views/Analysis/Chain/Chain.scss

```diff
@@ -0,0 +1,4 @@
+.chain {
+
```

Comment:
style is missing

Reply:
I have to refactor all the styles because now they are imported into the same file, but I think for now I will remove this file. Because it is not needed